test/e2e/serial/tests/configuration.go

```diff
@@ -693,9 +697,195 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
 				return kcCmNamesCur
 			}).WithContext(ctx).WithTimeout(5 * time.Minute).WithPolling(10 * time.Second).Should(Equal(kcCmNamesPre))
 		})
+
+		It("[reboot_required][slow] should be able to correctly identify topology manager policy without scheduler restarting", Label("reboot_required", "slow"), func() {
+			nroOperObj := &nropv1.NUMAResourcesOperator{}
```

Comment:
let's add a By() step here for fetching the matching kubeletconfig for the numaresources-operator CR

Reply:
got it